### ISSUE TITLE
log_level controls not functioning

### ISSUE THREAD

**Issue by andrewvaughan** [SergeyPirogov/webdriver_manager#375]
When setting `log_level` no effect is seen. I have tried both ways:

```python
os.environ['WDM_LOG_LEVEL'] = '0'

self._driver_cache = uc.Chrome(
    driver_executable_path = ChromeDriverManager(
                                print_first_line = False,
                                log_level        = 0
                             ).install(),
    options                = options,
    headless               = False
)
```

I still receive logs on `INFO` and on stdout:

```
[WDM] - ====== WebDriver manager ======
2022-05-19 14:23:36,684 [    INFO ] ====== WebDriver manager ====== (WDM)
[WDM] - Current google-chrome version is 101.0.4951
2022-05-19 14:23:36,860 [    INFO ] Current google-chrome version is 101.0.4951 (WDM)
[WDM] - Get LATEST chromedriver version for 101.0.4951 google-chrome
2022-05-19 14:23:36,860 [    INFO ] Get LATEST chromedriver version for 101.0.4951 google-chrome (WDM)
[WDM] - Driver [/Users/andrewvaughan/.wdm/drivers/chromedriver/mac64/101.0.4951.41/chromedriver] found in cache
2022-05-19 14:23:36,912 [    INFO ] Driver [/Users/andrewvaughan/.wdm/drivers/chromedriver/mac64/101.0.4951.41/chromedriver] found in cache (WDM)
```

I have tried setting `log_level` to `logging.DEBUG` as well, and it still outputs on `INFO`

**Comment by aleksandr-kotlyar** [comment]
log_level will be removed from constructor soon. Only env will left. If here is a bug it will be fixed soon.

**Comment by SergeyPirogov** [comment]
Fixed in v3.6.1
